decidim-core/app/cells/decidim/badge/show.erb

```diff
@@ -1,36 +1,14 @@
-<div class="column">
-  <div class="card card--badge absolutes badge-<%= badge.name %>">
-    <div class="right badge-tip">
-      <div data-tooltip data-position="top" title="<%= tooltip %>" data-yeti-box="<%= badge.name %>-tooltip" data-toggle="<%= badge.name %>-tooltip" data-resize="<%= badge.name %>-tooltip" class="has-tip" data-t="<%= badge.name %>-t">
-        <%= icon "info", role: "img", "aria-hidden": true %>
+<div data-badge="<%= badge.name %>">
+  <div class="profile__badge">
+    <div class="profile__badge-outline" style="--value:<%= status.level.to_f / 5 %>;<%= "--color:var(--primary)" if status.level.to_f / 5 == 1 %>">
+      <div class="profile__badge-circle">
+        <%= image_tag badge.image, alt: "#{badge_name} badge_image" %>
```

Comment:
Why do we have the ` badge_image` suffix in the alt text? I believe the `<img>` tag is already indicating it is an image, so we could just use the badge name as the alt text maybe?

At least don't leave it in the final version as it is currently.